waypoint/services/nats_service.py

```diff
@@ -40,7 +45,22 @@ async def _subscribe(
                     "subject": f"{NATS_SUBJECT}.*.{wallet_id}",
```

Comment:
PS, some cleanup we can do for this above:
- remove empty line after `if group_id`
- logs should be more accurate: we don't know if it's a tenant-admin call, we can just say: "Subscribe with group_id: {}", or "Subscribe without group_id". Plus those logs are probably unnecessary.
- define `subscribe_kwargs = {"stream": NATS_STREAM]}` once

Altogether:
```py
        try:
            logger.trace("Subscribing to JetStream for wallet_id: {}, group_id: {}", wallet_id, group_id}
            group_id = group_id or "*"
            subscribe_kwargs = {
                "stream": NATS_STREAM,
                "subject": f"{NATS_SUBJECT}.{group_id}.{wallet_id}",
            }
```